### ISSUE TITLE
unsupported function: HostHeader

### ISSUE THREAD

**Issue by NotSaifA** [htpcBeginner/docker-traefik#70]
I installed traefik, oauth, qbitorrent, but I can't seem to get Sonarr working. Here is the error I am getting in Traefik "error while parsing rule HostHeader(`sonarr.mydomain.com`): unsupported function: HostHeader"
 I am using the code specified in the example docker-compose file.

**Comment by NotSaifA** [comment]
I just found this update in the guide "Update July 10, 2020: Traefik 2.2.2 introduced a breaking change. I have temporarily changed the chevrotin tag to 2.2.1. If you want to use the latest version either add --global.insecureSNI to the CLI arguments below or change rule=Host(`subdomain.example.com`) to rule=HostHeader(`subdomain.example.com`) in the traefik labels for services. I will update the guide and GitHub repo at some point.". This managed to get rid of the error but I am still getting a 404 page not found error.

**Comment by NotSaifA** [comment]
I just found this update in the guide "Update July 10, 2020: Traefik 2.2.2 introduced a breaking change. I have temporarily changed the chevrotin tag to 2.2.1. If you want to use the latest version either add --global.insecureSNI to the CLI arguments below or change rule=Host(`subdomain.example.com`) to rule=HostHeader(`subdomain.example.com`) in the traefik labels for services. I will update the guide and GitHub repo at some point.". 

So basically if you are using 2.2.1 use the "Host" function, if you are on a higher version use "Hostheader"
